PracticeGrading.Data/Repositories/MeetingRepository.cs

```diff
@@ -0,0 +1,65 @@
+// <copyright file="MeetingRepository.cs" company="Maria Myasnikova">
+// Copyright (c) Maria Myasnikova. All rights reserved.
+// Licensed under the Apache-2.0 license. See LICENSE file in the project root for full license information.
+// </copyright>
+
+namespace PracticeGrading.Data.Repositories;
+
+using Microsoft.EntityFrameworkCore;
+using Entities;
+
+/// <summary>
+/// Сlass for interacting with the meeting entity.
+/// </summary>
+/// <param name="context"> Database context.</param>
+public class MeetingRepository(AppDbContext context)
+{
+    /// <summary>
+    /// Creates new meeting.
+    /// </summary>
+    /// <param name="meeting">New meeting.</param>
+    public async Task Create(Meeting meeting)
+    {
+        await context.Meetings.AddAsync(meeting);
+        await context.SaveChangesAsync();
+    }
+
+    /// <summary>
+    /// Updates meeting.
+    /// </summary>
+    /// <param name="meeting">Meeting to update.</param>
+    public async Task Update(Meeting meeting)
+    {
+        context.Meetings.Update(meeting);
+        await context.SaveChangesAsync();
+    }
+
+    /// <summary>
+    /// Gets meeting by id.
+    /// </summary>
+    /// <param name="id">Meeting id.</param>
+    /// <returns>Meeting.</returns>
+    public async Task<Meeting?> GetById(int id)
+    {
+        return await context.Meetings.FirstOrDefaultAsync(meeting => meeting.Id == id);
+    }
```

Comment:
```suggestion
    public async Task<Meeting?> GetById(int id)
        => await context.Meetings.FirstOrDefaultAsync(meeting => meeting.Id == id);
```
Не надо писать многословный код, если можно писать кратко.